certora/specs/LiquidityPoolMoneyFlow.spec

```diff
@@ -0,0 +1,157 @@
+// import "./ERC721Receiver.spec"; 
+// import "./EtherFiNodeInterface.spec"; 
+// import "./EtherFiNodesManagerSetup.spec"; 
+// import "./EigenLayerMethods.spec"; 
+import "./Basic.spec";
+
+// using LiquidityPool as Pool;
+using AuctionManager as auctionManager;
+
+// Status: All rules here pass
+// Run link: https://prover.certora.com/output/65266/e1ae0ebb78c346259a9bd46cf286b3bc/?anonymousKey=d2adcf131917ccf3f7512519f17a985f597b8ce3
+
+methods {
+    // dispatcher list for unresolved calls in
+    // EtherFiNode.withdrawFunds
+    unresolved external in EtherFiNode._ => DISPATCH [
+        CallHookHelper.HarnessCallHook()
+    ] default NONDET;
+
+    function CallHookHelper.HarnessCallHook() external with (env e) => CVLCallHook(e);
+
+}
+
+// This function is used to catch all cases where there is
+// a call outside the contract under verification and save
+// the target address whenever the value transferred is nonzero
+persistent ghost address money_recipient;
+function CVLCallHook(env e)  {
+    if (e.msg.value > 0) {
+        money_recipient = e.msg.sender;
+    }
+}
+
+// the only possible flow of validator funds is eigenpod -> etherfiNode ->
+// expectedParties where expected parties in the current contract are currently
+// bnftHolder / tnftHolder / nodeOperator / treasury. An upcoming change from @V
+// will be adjusting that mechanism slightly so that all funds flow directly to
+// the liquidity pool. But the important piece of the property to me is simply
+// that funds can't possibly be transferred to an unexpected target. This would
+// give the assurance that the worst case scenario of a logic bug is simply that
+// some validator funds could be temporarily stuck until we upgrade with a fix,
+// but user funds would never be lost.
+
+// * whichFunctionSendsETH_EtherFiNode in Basic.spec proves
```

Comment:
If we rely on some other rules in this verification, let's simply import them here with the `use rule` command.

Reply:
Ah. For some reason I either believed I ran these or assumed they were already verified but they were not. I adjusted them to get them passing. I think the adjustments to `Base.spec` are also fine, but take a look.